continuous_integration/environment-3.8-dev.yaml

```diff
@@ -5,8 +5,9 @@ channels:
 dependencies:
 - c-compiler
 - dask=2022.3.0
-- fastapi=0.69.0
+- fastapi=0.92.0
```

Comment:
However restricting to `0.92.0` results in [only one test failure]( https://github.com/dask-contrib/dask-sql/actions/runs/5685404699/job/15410230957?pr=1205#step:8:33 )

<details>

````python
_________________________________ test_routes __________________________________
[gw1] linux -- Python 3.8.17 /usr/share/miniconda3/envs/dask-sql/bin/python3.8

app_client = <starlette.testclient.TestClient object at 0x7fe5582847f0>

    def test_routes(app_client):
        assert app_client.post("/v1/statement", data="SELECT 1 + 1").status_code == 200
>       assert app_client.get("/v1/statement", data="SELECT 1 + 1").status_code == 405
E       TypeError: get() got an unexpected keyword argument 'data'

tests/integration/test_server.py:61: TypeError
````

</details>

Reply:
Fixed by PR ( https://github.com/dask-contrib/dask-sql/pull/1206 ). Marking resolved